doc-General_Configuration/topics/Configuration.adoc

```diff
@@ -93,7 +93,8 @@ To enable a cluster, host, or datastore for Capacity and Utilization Collection:
 
 [NOTE]
 =========================
-As new clusters, hosts, and datastores are discovered, you will need to come back to this configuration to enable collection of capacity and utilization data unless you have used the *Collect for All* check boxes.
+. As new clusters, hosts, and datastores are discovered, you will need to come back to this configuration to enable collection of capacity and utilization data unless you have used the *Collect for All* check boxes.
+. *Collect for All Clusters* must be checked to be able to collect C & U data from Cloud Providers such as Red Hat OpenStack or Amazon EC2.
```

Comment:
A few fixes:
- C & U data (as in above topic) --> I'd recommend spelling out as "capacity and utilization data", just to avoid acronyms. Looking through the guide we only abbreviate it when it's a UI label.
- Cloud Providers  should be lower case --> cloud providers
- I think Red Hat OpenStack is normally written as Red Hat OpenStack Platform, but we could probably also abbreviate it to just OpenStack like the rest of the procedure (not sure how crucial this is.)